### PR TITLE
update how we work section with new typography

### PR DESCRIPTION
**Pull request by omonk**
## Description - [Trello ticket](https://trello.com/c/vsJH7RoQ/132-revisit-a-few-more-updates-to-how-we-work-section-on-speciality-page)

Opening PR for deploy preview so James can take a look

### CODE REVIEW

File: src/templates/speciality/schemas.js [hunk 1]
```diff
@@ -196,16 +196,9 @@ const howWeWorkSchema = {
     },
     howWeWorkWithCopy: {
       type: 'string'
-    },
-    howWeWorkWithPractises: {
-      type: 'array'
     }
```
Comment (GBouffard) [inline src/templates/speciality/schemas.js]: Why did you remove `howWeWorkWithPractises`? It still should be in the query no?

File: src/components/Speciality/HowWeWorkWith.js [hunk 2]
```diff
@@ -20,35 +23,73 @@ const StyledRow = styled(Row)`
...
 
 const HowWeWorkWithSection = ({
-  howWeWorkWithTitle,
-  howWeWorkWithCopy,
-  howWeWorkWithPractises
+  howWeWorkWithTitle: title,
+  howWeWorkWithCopy: copy,
+  howWeWorkWithPractises: practises = []
 }) => {
+  const hasPractises = practises && practises.length
```
Comment (GBouffard) [inline src/components/Speciality/HowWeWorkWith.js]: I would move the `practises.length > 0` in this logic so you can use `hasPractises &&` below. It's nicer.

File: src/templates/speciality/schemas.js (hunk 1 above)
Comment (omonk) [reply src/templates/speciality/schemas.js]: I don't think we need it in here if its not required... 🤔 It wasn't working for me when I tried to just remove the practises in the `required` key

File: src/components/Speciality/HowWeWorkWith.js (hunk 2 above)
Comment (omonk) [reply src/components/Speciality/HowWeWorkWith.js]: @GBouffard nice spot, have updated
